goodkey/good_key.go

```diff
@@ -44,6 +44,16 @@ type KeyPolicy struct {
 	AllowECDSANISTP521 bool // Whether ECDSA NISTP521 keys should be allowed.
 }
 
+// NewKeyPolicy returns a KeyPolicy that allows RSA, ECDSA256 and ECDSA384.
+func NewKeyPolicy() KeyPolicy {
+	return KeyPolicy{
+		AllowRSA:           true,
+		AllowECDSANISTP256: true,
+		AllowECDSANISTP384: true,
+		AllowECDSANISTP521: false,
```

Comment:
@jsha any reason not to just remove this and the related code in the switch statement below?